tests/fixtures_peak_detection.py

```diff
@@ -9,3 +12,22 @@ def fixture_maiden_voyage_data():
     return _run_peak_detection(
         "maiden_voyage_data.tsv", sampling_rate_construct=200, flip_data=True
     )
+
+
+@pytest.fixture(scope="session", name="noisy_data_A1")
+def fixture_noisy_data_A1():
+    return _run_peak_detection(
+        os.path.join("noisy_data", "noisy_data_A1.tsv"),
+        sampling_rate_construct=600,
+        flip_data=True,
+    )
+
+
+@pytest.fixture(scope="session", name="noisy_data_B1")
+def fixture_noisy_data_B1():
```

Comment:
are these "noisy_data.tsv" the files from Travis?  If so, I apologize, I thought when you asked me about how Samir got the TSV data I didn't make the connection that you were about to do the same thing.  Pulling out the data as TSV was something that he did at the very beginning, but then we determined that it would just be better to include the whole H5 file because it has relevant metadata about the experiment.  That way if down the road one recording in our test set is being particularly troublesome to analyze with our peak finding algorithm, we can review the metadata and determine if it's possible that that's just a really weird dataset (due to odd biological conditions or hardware conditions in the alpha/beta) that we don't think we need to support

Reply:
If these are Travis' data, please include the H5 files and not TSV format